spec/services/applications/accept_spec.rb

```diff
@@ -372,6 +370,17 @@
         let(:new_course) { create(:course, :sl, course_group: new_course_group) }
```

Comment:
I think you'll want to rebase before merging as these course identifiers have changed to remove the abbreviations. We can also drop the find on the course group creation above now as we use `initialize_with`, so `create(:course_group, name: "specialist")` will do a find/create for us.

It took me a minute to figure out what we're testing here; would it make sense to rename `new_course/group` to something like `wrong_course/group`?